src/riscv/linux/hwcap.c

```diff
@@ -0,0 +1,37 @@
+#include <riscv/linux/api.h>
+
+#if CPUINFO_MOCK
+    #include <cpuinfo-mock.h>
+#endif
+
+#if CPUINFO_ARCH_RISCV || CPUINFO_ARCH_RISCV64
+    #include <sys/auxv.h>
+#else
+    #define AT_HWCAP 16
+    #define AT_HWCAP2 26
+#endif
+
+#if CPUINFO_MOCK
+    static uint32_t mock_hwcap = 0;
+    void cpuinfo_set_hwcap(uint32_t hwcap) {
+        mock_hwcap = hwcap;
+    }
+
+	static uint32_t mock_hwcap2 = 0;
```

Comment:
Looks like this is indented with a tab character and the above is indented with spaces.